samples/psa/its/overlay-secure_storage.conf

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_MBEDTLS=y
+CONFIG_MBEDTLS_PSA_CRYPTO_C=y
```

Comment:
what's the purpose of Mbed TLS here? Unless I've missed something it's not used in the sample